src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -168,7 +168,32 @@ private void HandleClusterDatabaseChanged(string databaseName, long index, strin
             }
         }
 
-        private void UnloadDatabase(string databaseName)
+        private void UnloadDatabase(string databaseName, bool dbRecordIsNull = false)
+        {
+            if (dbRecordIsNull)
+            {
+                UnloadDatabaseInternal(databaseName);
```

Comment:
Need a description here explaining the situation

Reply:
there is a comment on line 73, I can also add it here.